java/Options.java

```diff
@@ -0,0 +1,12 @@
+package io.github.f3d_app.f3d;
+
+public class Options {
+
+    public Options(long nativeAddress) {
+        mNativeAddress = nativeAddress;
+    }
+
+    public native void toggle(String name);
```

Comment:
looks like you did not expose the whole public API, is that intentionnal ?

Reply:
Yes, I just need to add all of them, but I will do it in another PR later. The interactor is not bound yet too. I just bound what I needed for Android

Reply:
I just wanted to create the foundation for the Java bindings in this PR.

Reply:
fair enough, ideally keep an issue alive with what would be needed next.

Reply:
https://github.com/f3d-app/f3d/issues/426